<launch>
    <rosparam file="$(find armin_state_publisher_from_gui)/config/hardware.yaml" command="load"/>
    <rosparam file="$(find armin_state_publisher_from_gui)/config/controllers.yaml" command="load"/>
    <rosparam file="$(find armin_state_publisher_from_gui)/config/joint_limits.yaml" command="load"/>
    <node name="armin_hardware_interface" pkg="armin_state_publisher_from_gui" type="armin_hardware_interface_node" output="screen"/>
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/"
        args="
            /armin/controller/state
            /armin/controller/position/armin_position
        "/>
</launch>
